src/main/java/org/terracotta/build/plugins/CustomHostPlugin.java

```diff
@@ -0,0 +1,76 @@
+package org.terracotta.build.plugins;
+
+import org.gradle.api.DefaultTask;
+import org.gradle.api.Plugin;
+import org.gradle.api.Project;
+import org.gradle.api.file.DirectoryProperty;
+import org.gradle.api.file.RegularFile;
+import org.gradle.api.file.RegularFileProperty;
+import org.gradle.api.provider.ListProperty;
+import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.OutputFile;
+import org.gradle.api.tasks.TaskAction;
+import org.gradle.api.tasks.TaskProvider;
+import org.gradle.api.tasks.testing.Test;
+
+import javax.inject.Inject;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singleton;
+import static org.terracotta.build.PluginUtils.capitalize;
+
+public class CustomHostPlugin implements Plugin<Project> {
```

Comment:
Since this plugin is now constrained to just "localhost" modifications I wonder if we should be renaming things to reflect that to avoid confusion.

Reply:
changed to CustomLocalHostPlugin